pkgs/servers/minio/default.nix

```diff
@@ -1,14 +1,13 @@
 { lib, stdenv, fetchurl, go }:
 
 stdenv.mkDerivation rec {
-  name = "minio-${shortVersion}";
+  name = "minio-${version}";
 
-  shortVersion = "20170613";
-  longVersion = "2017-06-13T19-01-01Z";
+  version = "2017-09-29T19-16-56Z";
```

Comment:
What a weird version scheme. Only milliseconds are missing.

Reply:
I guess it allows for a pretty rapid release cycle 😉